frontend/wallet/src/render/components/WalletSettings.re

```diff
@@ -0,0 +1,79 @@
+open Tc;
+
+module Styles = {
+  open Css;
+
+  let container = style([padding(`rem(1.0))]);
+
+  let backHeader = style([display(`flex), alignItems(`center)]);
+
+  let backIcon =
+    style([
+      color(Theme.Colors.hyperlinkAlpha(1.0)),
+      hover([color(Theme.Colors.hyperlinkAlpha(0.5))]),
+    ]);
+
+  let backHeaderText =
+    merge([Theme.Text.Header.h3, style([color(Theme.Colors.midnight)])]);
+
+  let label =
+    merge([
+      Theme.Text.Body.semiBold,
+      style([
+        color(Theme.Colors.midnight),
+        margin(`rem(0.25)),
+        marginTop(`rem(1.0)),
+      ]),
+    ]);
+
+  let textBox = style([maxWidth(`rem(21.))]);
+};
+
+[@react.component]
+let make = (~publicKey) => {
+  let (addressBook, updateAddressBook) =
+    React.useContext(AddressBookProvider.context);
+  <div className=Styles.container>
+    <div className=Styles.backHeader>
+      <span
+        className=Styles.backIcon
+        onClick={_ => ReasonReact.Router.push("/settings")}>
```

Comment:
does ReasonReact.Router have a `history.goBack()` or `pop()` method?

Reply:
We can do it if we need to yeah